core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -2228,7 +2253,7 @@ taskOverlap: ALLOW_OVERLAPPING_EXECUTION EQ trueFalse
 sql: EXECUTE IMMEDIATE DBL_DOLLAR | sqlCommand | call
     ;
 
-call: CALL objectName L_PAREN exprList? R_PAREN
+call: CALL objectName L_PAREN exprList? R_PAREN SEMI
```

Comment:
```suggestion
call: CALL objectName L_PAREN exprList? R_PAREN
```
don't do that. it'll break BC compat
<img width="663" alt="image" src="https://github.com/user-attachments/assets/cbb1e953-47e7-44ef-b734-ab30d82698e4">